### ISSUE TITLE
Support different YAML/JSON styles

### ISSUE THREAD

**Issue by KevinSayers** [nextflow-io/cwl2nxf#8]
Currently the converter does not handle the CWL style shown below, only CWL that lacks - id will parse correctly. Also the input file can only be in YAML, JSON is unsupported presently. 

```
- id: fooInput
```

**Comment by pditommaso** [comment]
i'm still not understanding if there's an automated conversion/normalisation tool between these two formats. Could you please investigate with the CWL community? 

Also create a separate issue for the missing JSON input handling.

**Comment by KevinSayers** [comment]
There is an automated conversion from YAML to JSON, but all current parsing is based on the references for YAML. 

I think we can easily get around this though by checking if
```
- id: fooID
``` 
is present in the lines from a file, and then just doing a replace

```
cwlline.replace('- id:', '')
```

**Comment by pditommaso** [comment]
No, I mean an automated conversion between the different versions of the YAML format.
